eng/.docsettings.yml

```diff
@@ -6,7 +6,7 @@ omitted_paths:
   - sdk/*/azure-mgmt*/*
   - sdk/keyvault/azure-keyvault/*
   - doc/*
-  - sdk/*/samples/*
+  - sdk/**/samples/*
```

Comment:
@scbedd Did we actually need the double `*`? I was under the impression a single star should work and we have other places were we are using a single star.

Reply:
Yeah. Definitely. Due to the fact we're including multiple directories.

Reply:
This is the path we're glob matching.

`sdk/cosmos/azure-cosmos/samples/README.md` 

Original glob pattern will only match match `sdk/<any service/samples/README.md`